packages/logger-pretty/lib/LoggerPretty.ts

```diff
@@ -1,5 +1,5 @@
-import { inspect } from 'util';
 import { Logger } from '@comunica/types';
+const inspect = require('object-inspect');
```

Comment:
This one should have type definitions: https://www.npmjs.com/package/@types/object-inspect
Could you check if those work, so that this can become an import statement?

Reply:
Done!